### PR TITLE
Added example on how to provision a cluster

### PR DESCRIPTION
**Pull request by abergman**
I've created a simple example in PHP on how one can build an cluster using GleSYS loadbalancer. The cluster can contain N servers.

### CODE REVIEW

File: PHP/CreateCluster/php-restclient-master/LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2011 Travis Dent <tcdent@gmail.com>
```
Comment (emil-nasso) [inline PHP/CreateCluster/php-restclient-master/LICENSE]: Who is travis dent? :)

**Comment by emilv** [comment]
php-restclient-master doesn't seem especially well suited for this API. You still need to concatenate all arguments to the URL manually. So what does the library provide?

**Comment by emilv** [comment]
Otherwise nice example!

**Comment by wecc** [comment]
I've promised @abergman to take a look at this sooner, it's a great example but I think we can simplify it. Thanks for giving this some attention @emilv!

**Pull request closed without merging**